### PR TITLE
Add FindFaces sample code

### PR DESCRIPTION
**Pull request by zhizhoualan**
## Purpose

Add FindFaces sample code
- ...

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: sample code
```

## How to Test

- Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

- Test the code

```

```

## What to Check

Verify that the following are valid

- ...

### CODE REVIEW

**Review by geos-ms — COMMENTED** [review]
Editorial feedback: detect faces, retrieve bounding box and face ID

File: face/FindFaces/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+## Find Faces
+
+This sample demonstrates how to create and manage face collections, add faces, and verify faces within images using the Azure Face API.
+
+### Key Features
+
+* Learn how to use the Azure Face API to detect faces and retrieve bounding box and face ID.
```
Comment (geos-ms) [inline face/FindFaces/README.md:8]: Editorial feedback: detect faces, retrieve bounding box and face ID.

File: face/FindFaces/python/add_face_to_collection.ipynb [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    "            else:\n",
+    "                print(\"No Operation-Location header found in the response.\")\n",
+    "        else:\n",
+    "            return f\"Failed to add face: {response.json()}\"\n",
+    "\n",
+    "# Example usage\n",
+    "FACE_KEY = os.environ[\"FACE_API_KEY\"]\n",
+    "FACE_ENDPOINT = os.environ[\"FACE_ENDPOINT_URL\"]\n",
+    "image_path = 'path_to_image.jpg'\n",
+    "target_type = 'face_list'  # or 'large_face_list', 'person_group', 'large_person_group', or 'person_directory'\n",
```
Comment (der3318) [inline face/FindFaces/python/add_face_to_collection.ipynb:138]: Q: Do we really want to support "large_face_list", "large_person_group" or " person_directory" here? These collections are not demonstrated in the other three sample.

File: face/FindFaces/python/find_faces_in_collection.ipynb [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    "    def detect_face(image_path):\n",
+    "        with FaceClient(endpoint, AzureKeyCredential(subscription_key), headers = {\"x-ms-useragent\": \"find-faces-in-collection/1.0.0\"} ) as face_client:\n",
+    "            with open(image_path, 'rb') as image_data:\n",
+    "                detected_faces = face_client.detect(\n",
+    "                        image_content=image_data,\n",
+    "                        detection_model=FaceDetectionModel.DETECTION_03,\n",
+    "                        recognition_model=FaceRecognitionModel.RECOGNITION_04,\n",
+    "                        return_face_id=True,\n",
+    "                    )\n",
+    "            return [face['faceId'] for face in detected_faces]\n",
```
Comment (der3318) [inline face/FindFaces/python/find_faces_in_collection.ipynb:55]: While the returned `FaceDetectionResult` object does allow syntax sugar `__getitem__` to access/lookup underlying JSON response, SDK owners suggest to retrieve the structed info via predefined instance variable.

```suggestion
    "                detected_faces = face_client.detect(\n",
    "                        image_content=image_data.read(),\n",
    "                        detection_model=FaceDetectionModel.DETECTION_03,\n",
    "                        recognition_model=FaceRecognitionModel.RECOGNITION_04,\n",
    "                        return_face_id=True,\n",
    "                    )\n",
    "            return [face.face_id for face in detected_faces]\n",
```

File: face/FindFaces/python/add_face_to_collection.ipynb (hunk 2 above)
Comment (zhizhoualan) [reply face/FindFaces/python/add_face_to_collection.ipynb:138]: > Q: Do we really want to support "large_face_list", "large_person_group" or " person_directory" here? These collections are not demonstrated in the other three sample.

Good question. This sample code is separate and will be shared with our customers who encountered error 400 when adding a face to our face collection. Therefore, it's better to support all related APIs. Please let me know your thoughts. Thanks.

Comment (der3318) [reply face/FindFaces/python/add_face_to_collection.ipynb:138]: Thanks for the context. I’m with you on that.

**Review by der3318 — APPROVED** [review]

**Review by geos-ms — APPROVED** [review]

**Pull request merged**